src/instructions/proctored_exam/SubmitProctoredExamInstructions.jsx

```diff
@@ -49,19 +49,20 @@ const SubmitProctoredExamInstructions = () => {
           />
         </p>
       )}
-      <Button variant="primary" onClick={submitExam}>
+      <Button variant="primary" onClick={submitExam} className="mr-2" data-testid="end-exam-button">
         <FormattedMessage
           id="exam.SubmitProctoredExamInstructions.submit"
           defaultMessage="Yes, end my proctored exam"
         />
       </Button>
-      &nbsp;
```

Comment:
Did you missed `&nbsp;` on purpose?

Reply:
Yes, I added right margin to the button above instead, I think it is better than adding spaces
I can revert this if you think we should use &nbsp;